packages/lab/src/file-drop-zone/FileDropZone.tsx

```diff
@@ -1,115 +1,36 @@
-  forwardRef,
-  HTMLAttributes,
-  KeyboardEvent,
-  ReactNode,
-  SyntheticEvent,
-  useCallback,
-  useEffect,
-  useRef,
-  useState,
-} from "react";
-import { containsFiles, extractFiles, validateFiles } from "./internal/utils";
-import { FilesValidator } from "./validators";
+import { DragEventHandler, forwardRef, HTMLAttributes, useState } from "react";
+import { containsFiles } from "./internal/utils";
 
 import { useWindow } from "@salt-ds/window";
 import { useComponentCssInjection } from "@salt-ds/styles";
 
 import fileDropZoneCss from "./FileDropZone.css";
 
-// Recommended button label by ADA review
-const buttonLabel = "Browse files";
-const INVALID_DROP_TARGET = "Drop target doesn't contain any file.";
-
-export type FilesAcceptedEventHandler = (
-  files: ReadonlyArray<File>,
-  event: SyntheticEvent
-) => void;
-
-export type FilesRejectedEventHandler<ErrorType = string> = (
-  errors: ReadonlyArray<ErrorType>,
-  event: SyntheticEvent
-) => void;
-
-/**
- * Removed deprecated props
- *
- * - onDrop
- * - onFileTypeError
- * - showUploadButton
- */
-
 export interface FileDropZoneProps extends HTMLAttributes<HTMLDivElement> {
-  /**
-   * `accept` attribute for HTML <input>.
-   *
-   * A comma separated list of file types the user can pick from the file input dialog box.
-   */
-  accept?: string;
-
-  /**
-   * The text content of the drop area component.
-   */
-  children?: ReactNode;
-
-  /**
-   * Additional usage information.
-   */
-  description?: string;
-
   /**
    * If `true`, the file drop zone will be disabled.
    */
   disabled?: boolean;
-
   /**
-   * This prop is used to help implement the accessibility logic.
-   * If you don't provide this prop. It falls back to a randomly generated id.
+   * Status indicator to be displayed.
    */
-  id?: string;
-
-  /**
-   * Callback on successful file drop or selection.
-   */
-  onFilesAccepted?: FilesAcceptedEventHandler;
-
-  /**
-   * @see `validate` prop
-   * Callback on drop or input in case of an error. A list of errors will be provided as input.
-   */
-  onFilesRejected?: FilesRejectedEventHandler;
-
-  /**
-   * A list of custom validation functions. Every function is provided with the entire file list as input
-   * thus can perform validations on all files. Each function needs to return one or more errors in case of
-   * a failed validation, or `undefined` in case of a successful one.
-   *
-   * All errors are collected in the end and returned as an array to `onFilesRejected`.
-   */
-  validate?: ReadonlyArray<FilesValidator<any>>;
+  status?: "success" | "error";
 }
 
 const withBaseName = makePrefixer("saltFileDropZone");
 
 export const FileDropZone = forwardRef<HTMLDivElement, FileDropZoneProps>(
   function FileDropZone(
     {
-      accept,
+      status,
       className,
       children,
-      description,
       disabled,
-      id: idProp,
-      validate,
-      onFilesAccepted,
-      onFilesRejected,
+      onDragOver: onDragOverProp,
```

Comment:
Do we need to rename here? This doesn't follow our current pattern since it's not already present.